phpunit/class-extend-preload-paths-test.php

```diff
@@ -7,20 +7,48 @@
 
 class Extend_Preload_Paths_Test extends WP_UnitTestCase {
 	/**
-	 * Tests '/wp/v2/blocks' added if missing.
+	 * Post object.
+	 *
+	 * @var WP_Post
+	 */
+	protected static $post;
+
+	public static function wpSetUpBeforeClass( $factory ) {
+		self::$post = $factory->post->create_and_get();
+	}
+
+	/**
+	 * Tests paths added if missing.
 	 */
 	function test_localizes_script() {
-		$preload_paths = gutenberg_extend_block_editor_preload_paths( array() );
+		$post = (object) array(
```

Comment:
Is this variable (and in the test below) unused?

Reply:
Yep, I made a mistake leaving that there after discovering that I could use the post factory in the test. Thanks for pointing it out.